packages/block-editor/src/components/block-controls/slot.native.js

```diff
@@ -14,19 +10,18 @@ import warning from '@wordpress/warning';
 import groups from './groups';
 
 export default function BlockControlsSlot( { group = 'default', ...props } ) {
-	const accessibleToolbarState = useContext( ToolbarContext );
 	const Slot = groups[ group ]?.Slot;
 	if ( ! Slot ) {
 		warning( `Unknown BlockControls group "${ group }" provided.` );
 		return null;
 	}
 
 	if ( group === 'default' ) {
-		return <Slot { ...props } fillProps={ accessibleToolbarState } />;
+		return <Slot { ...props } />;
 	}
 
 	return (
-		<Slot { ...props } fillProps={ accessibleToolbarState }>
+		<Slot { ...props }>
```

Comment:
I encountered an error when testing the native version due to not passing `fillProps` here. Seems this is caused because the default value set by the `Slot` component is an empty object, while the `fill` of `BlockControls` component expects `fillProps` to be an array.

https://github.com/WordPress/gutenberg/blob/80678f95b8ed4ea13e6b692dd614bdc4385bdce0/packages/components/src/slot-fill/slot.js#L63

A quick solution would be to pass an empty array:
```
const EMPTY_ARRAY = [];

export default function BlockControlsSlot( { group = 'default', ...props } ) {
...
	if ( group === 'default' ) {
		return <Slot { ...props } fillProps={ EMPTY_ARRAY } />;
	}

	return (
		<Slot { ...props } fillProps={ EMPTY_ARRAY }>
```

Reply:
As a side note, this is also causing the [job `Unit Tests / Mobile`](https://github.com/WordPress/gutenberg/actions/runs/5196809076/jobs/9370894497?pr=51264) to fail.

Reply:
Looking at this failure now :+1:

Reply:
I fixed it by doing something we'd probably have to do anyway. The array of forwarded context is no longer the `fillProps` value _itself_, but a `fillProps.forwardedContext` field. That makes it possible to also pass something else along in `fillProps`.

The bug was that `fillProps` are defaulted to `{}`, so my code: `( fillProps = [] ) => ...` never defaulted to array, and instead wanted to use `{}` as array.

Everything is 💚 again after this.

Reply:
I tested again with the fix and confirmed that works as expected 🎊 .